airbyte-integrations/bases/connector-acceptance-test/unit_tests/test_asserts.py

```diff
@@ -136,7 +131,7 @@ def test_verify_records_schema(configured_catalog: ConfiguredAirbyteCatalog):
 )
 def test_verify_records_schema_with_fail_on_extra_columns(configured_catalog: ConfiguredAirbyteCatalog, json_schema, record, should_fail):
     """Test that fail_on_extra_columns works correctly with nested objects, array of objects"""
-    configured_catalog.streams[0].stream.json_schema =json_schema
```

Comment:
Out of curiosity, what formatter do you personally use, and what formatter does most of the team have? 

@alafanechere works on setting up requirements on new PRs including type checks with mypy, and I'm considering a formatter as well.

Reply:
> what formatter do you personally use?
Mainly simple airbyte-ci format fix all (but it is needed to remove unit_test folder from excluded list in `pyproject.toml`

Reply:
@natikgadzhi we already have a repo wide formatter (`airbyte-ci format fix all/python/js/java`) which runs in CI.
We can double the same rules at the connector level if we want to localize the formatting.